config.go

```diff
@@ -249,6 +250,8 @@ func xmlToFileLogWriter(filename string, props []xmlProperty, enabled bool) (*Fi
 			daily = strings.Trim(prop.Value, " \r\n") != "false"
 		case "rotate":
 			rotate = strings.Trim(prop.Value, " \r\n") != "false"
+		case "integersuffix":
+			integerSuffix = strings.Trim(prop.Value, " \r\n") != "false"
```

Comment:
I changed the default behavior within the library, but planned to leave the old behavior as default in the agent. So there shouldn't be a backwards-incompatible change, but new uses of log4go will use the preferred behavior. In other words, if a user upgraded an agent with no other changes, they *would* see the same behavior.

One of the changes here plus a forthcoming PR will actually tolerate rotation failure, so even if an integer suffix is used and we reach the 1000th rotation, log4go will continue to write to the "current" logfile.